README.md

```diff
@@ -33,29 +36,31 @@ The theme can be built using the following commands:
 $ npm install
 $ npm run build
 ```
-The theme will then be built at `/govuk`, ready for use in the local Keycloak dev server.
+The theme will then be built at `/govuk`, ready for use in the local Keycloak server.
 
-### Local Keycloak dev server
-A local Keycloak server for developing the theme is available as a Docker container.
+### Local Keycloak server
+A local Keycloak server for developing and testing the theme is available as a Docker container.
 
 To bring up the server, make sure you’ve installed and started [Docker Community Edition](https://docs.docker.com/engine/installation/), then use the following commands:
 ```bash
 $ npm install
 $ npm run build
 $ docker-compose up
 ```
-The Keycloak server will now be available on <http://localhost:8080>.
+The Keycloak server will now be available on <http://localhost:8080>. You can log into the Administration Console using “**admin**” as both username and password.
 
-To enable the theme in the local Keycloak server, log in (using “**admin**” as both username and password), and follow steps 2 and 3 in [the “Using” section above](#using).
+<img alt="" src="docs/images/keycloak-local.png" width="512" height="300">
```

Comment:
I think there's markdown code for this.
![one does not simply write markdown](https://imgflip.com/s/meme/One-Does-Not-Simply.jpg)